stdlib/priorityqueue.gr

```diff
@@ -220,12 +207,13 @@ provide let drain = pq => {
  * and a negative number if the first has less priority.
  *
  * @param array: An array of values used to initialize the priority queue
- * @param comp: A comparator function used to assign priority to elements
+ * @param comp: A comparator function used to assign priority to elements (default is `compare` pervasive)
  * @returns A priority queue containing the elements from the array
  *
  * @since v0.5.4
+ * @history v0.6.0: Made `comp` a default argument
  */
-provide let fromArray = (array, comp) => {
+provide let fromArray = (array, comp=compare) => {
```

Comment:
Let's rename all these "comp" to compare

Reply:
You pointing this out also brought to my attention that the sort comparator could also be optional